frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -108,12 +110,21 @@ const Map: React.FC<MapProps> = ({
 
   const handleViewportChange = () => {
     const bounds = getBounds();
-    const { address, municipality, agencies, classificationId, minLotSize, maxLotSize } = mapFilter;
+    const {
```

Comment:
Just nitpicking, remove whitespace

Reply:
On it!

Reply:
Looks like this is auto-formatted. Going to dismiss for now